### PR TITLE
[AIRFLOW-6111] [AIP-21]  Rename GCP spanner operator and hook

### PR DESCRIPTION
**Pull request by michalslowikowski00**
PR contains changes regarding AIP-21 (renaming GCP operators and hooks):

renamed GCP modules
adde deprecation warnings to the contrib modules
fixed tests
updated UPDATING.md

---
- [x] Description above provides context of the change
- [x] Commit message contains [\[AIRFLOW-6111\]](https://issues.apache.org/jira/browse/AIRFLOW-6111) or `[AIRFLOW-XXXX]` for document-only changes
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: UPDATING.md [hunk 1]
```diff
@@ -518,16 +518,16 @@ The following table shows changes in import paths.
...
+|airflow.contrib.operators.gcp_natural_language_operator.CloudLanguageAnalyzeEntitiesOperator                      |airflow.providers.google.cloud.operators.natural_language.CloudLanguageAnalyzeEntitiesOperator                                |
+|airflow.contrib.operators.gcp_natural_language_operator.CloudLanguageAnalyzeEntitySentimentOperator               |airflow.providers.google.cloud.operators.natural_language.CloudLanguageAnalyzeEntitySentimentOperator                         |
+|airflow.contrib.operators.gcp_natural_language_operator.CloudLanguageAnalyzeSentimentOperator                     |airflow.providers.google.cloud.operators.natural_language.CloudLanguageAnalyzeSentimentOperator                               |
+|airflow.contrib.operators.gcp_natural_language_operator.CloudLanguageClassifyTextOperator                         |airflow.providers.google.cloud.operators.natural_language.CloudLanguageClassifyTextOperator                                   |
```
Comment (turbaszek) [inline UPDATING.md]: I think this are unrelated changes?

**Review by turbaszek — CHANGES_REQUESTED** [review]
LGTM just unrelated changes in UPDATING.md

File: UPDATING.md (hunk 1 above)
Comment (michalslowikowski00) [reply UPDATING.md]: What about new paths?

Comment (turbaszek) [reply UPDATING.md]: Yeah, but this changes `CloudLanguage` and the PR says `Spanner` 💃

Comment (michalslowikowski00) [reply UPDATING.md]: Ops...  Fix is on the way.

**Review by turbaszek — APPROVED** [review]
